src/main/java/constant/HttpStatus.java

```diff
@@ -0,0 +1,22 @@
+package constant;
+
+public enum HttpStatus {
+    OK ("200", "OK"),
+    CREATED ("201", "Created"),
+    REDIRECT ("302", "Found"),
+    BAD_REQUEST ("400", "Bad Request"),
+    NOT_FOUND ("401", "Not Found"),
```

Comment:
401과 404가 섞여 있네요 ㅎㅎ.
```suggestion
    NOT_FOUND ("404", "Not Found"),
```